posthog/kafka_client/client.py

```diff
@@ -216,7 +216,5 @@ def __init__(self):
     def produce(self, sql: str, topic: str, data: Dict[str, Any], sync: bool = True):
```

Comment:
The `sync` parameter is unused here now. But some paths that call this still care about `sync`... should they not any more?

Reply:
async_execute always called sync_execute :shrug: 

We could remove the param, but given this only affects testing :shrug:

Reply:
> async_execute always called sync_execute 🤷

🤣 

👍